app/views/posts/edit.html.erb

```diff
@@ -0,0 +1,6 @@
+<%= form_for @post do |form| %>
+  <%= form.label :message %>
+  <%= form.text_field :message %>
+
+  <%= form.submit "Submit" %>
+<% end %>
```

Comment:
The symbol on the end of this section is telling you that you should have one blank line at the end of the file.  This blank line is conventionally used to show that the file is complete, rather than truncated for some reason in error.

Thought you might like to know that :)